### ISSUE TITLE
[BUG/Help] <title>测试集数据超过1万条后，推理结果为空

### ISSUE THREAD

**Issue by situjunhao** [THUDM/ChatGLM-6B#980]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

第10001条数据，{label:"xxxxxxxxxxxxxxx", predict:""}

### Steps To Reproduce

正常跑predict方法即可，测试数据超过1万。

### Environment

```markdown
- OS:
- Python:
- Transformers:
- PyTorch:
- CUDA Support (`python -c "import torch; print(torch.cuda.is_available())"`) :
```
